packages/codepen-loader/just.config.js

```diff
@@ -16,4 +30,4 @@ task(
   )
 );
 
-task('build', series('clean', 'copy', parallel('jest', 'ts')));
+task('build', series('clean', 'copy', parallel('ts', condition('jest', () => !argv().min))));
```

Comment:
honestly think that we should have a separate command for jest :)

task('build', ...)
task('test', ...)

not for this fix, of course, but just a thought